README.md

```diff
@@ -481,6 +491,19 @@ $ ls -al samples/maven-repo/org/mozilla/rust-android-gradle/org.mozilla.rust-and
 samples/maven-repo/org/mozilla/rust-android-gradle/org.mozilla.rust-android-gradle.gradle.plugin/0.4.0/org.mozilla.rust-android-gradle.gradle.plugin-0.4.0.pom
 ```
 
+# Testing Local changes
+After running `./gradlew publish` -- An easy way to locally test changes made in this plugin is to simply add this to your project's `settings.gradle`:
```

Comment:
You needn't run `./gradlew publish` to use this, in fact.  The plugin will be built (and updated!) as needed.